### ISSUE TITLE
UsePreviousValue Incompatible with Create

### ISSUE THREAD

**Issue by ingalls** [mapbox/cfn-config#182]
### Context

The [NoEcho PR](https://github.com/mapbox/cfn-config/pull/178/files) does not differentiate between an `update` and a `create` operation. Upon creating a new stack, the `UsePreviousValue: True` can be set. Once submitted to Cloudformation, this will be rejected with the following error:

```
Create failed: Failed to generate changeset: ValidationError: Cannot specify UsePreviousValue on Parameters when ChangeSetType is CREATE
```

cc/ @rclark @danmactough

**Comment by rclark** [comment]
Thank you for finding these bugs! I'll add a fix in #181 as soon as I can.

**Comment by miccolis** [comment]
:wave: I'd run into this and am seeing it fixed w/ 3.0.1

Thanks @rclark & @ian29!

**Comment by ian29** [comment]
hooray! also: hello everyone. 

fixed in #181
